pkg/pillar/cmd/hardwaremodel/hardwaremodel.go

```diff
@@ -25,7 +26,7 @@ type info interface {
 	Class() string
 }
 
-func hwFp() {
+func hwFp(outputFile string) {
```

Comment:
Can we do this as a small separate PR?

Reply:
https://github.com/lf-edge/eve/pull/1319